### PR TITLE
add scala 2.13.15

### LINKED ISSUE #876: Add build for scala 2.13.15

### PR DESCRIPTION
**Pull request by drewfeelsblue**
Fixes #876.

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -20,7 +20,7 @@ developers := List(
 )
 
 scalaVersion := "3.5.1"
-crossScalaVersions := Seq("2.12.19", "2.12.20", "2.13.13", "2.13.14", "3.3.3", "3.5.1")
+crossScalaVersions := Seq("2.12.19", "2.12.20", "2.13.13", "2.13.14", "2.13.15", "3.3.3", "3.5.1")
```
Comment (saeltz) [inline build.sbt:23]:
```suggestion
crossScalaVersions := Seq("2.12.19", "2.12.20", "2.13.14", "2.13.15", "3.3.3", "3.5.1")
```

**Review by saeltz — COMMENTED** [review]
Thanks for contributing!

Would you please also change the `.github/workflows/pr-checks.yml`? See https://github.com/scapegoat-scala/scapegoat/pull/869/files for inspiration.

File: build.sbt (hunk 1 above)
Comment (drewfeelsblue) [reply build.sbt]: fixed

**Comment by saeltz** [comment]
Thanks, @drewfeelsblue!

@mccartney, would you issue a new release please?

**Pull request merged**